apis/core/v1alpha1/field_export.go

```diff
@@ -24,6 +24,7 @@ type FieldExportTarget struct {
 	// Namespace is marked as optional, so we cannot compose `NamespacedName`
 	Namespace *string               `json:"namespace,omitempty"`
 	Kind      FieldExportOutputType `json:"kind"`
+	Key       *string               `json:"key,omitempty"`
```

Comment:
Go ahead and add a docstring above here so that our API reference docs will include an explanation for the field.

Reply:
Added docs. a974e46b904f7299f756fcd0a90d5a66522859fb